extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -27,11 +28,22 @@ export function trimUri(innerUri: vscode.Uri, outerUri: vscode.Uri): string {
 	let innerParts = innerUri.path.split('/');
 	let outerParts = outerUri.path.split('/');
 
+	if (path.isAbsolute(outerUri.path)
+		&& innerParts.length > 0 && outerParts.length > 0
+		&& innerParts[0].toLowerCase() !== outerParts[0].toLowerCase()) {
+		throw new Error(constants.ousiderFolderPath);
```

Comment:
This error message explicitly mentions project file, but this util function gets used on more than just that, I think?  Maybe this specific error message be higher up (like in `readProjFile()`) after this line throws or this message should be more generic.

Reply:
I changed the msg a bit but also took a quick look at places where its used... most are with existing file operations so its okay to refer to project file as such.